docs/userguide/networking/dockernetworks.md

```diff
@@ -1,538 +0,0 @@
-<!--[metadata]>
```

Comment:
i should have done a git mv. i did a FS rename. is that fixable? rename it back and then git mv?